### PR TITLE
Add basic LWJGL/app window

### PR DESCRIPTION
**Pull request by apostrophest**

### CODE REVIEW

File: src/main/java/com/github/reisnera/gameboylfb/App.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+import org.lwjgl.glfw.GLFWErrorCallback;
+import org.lwjgl.glfw.GLFWKeyCallback;
+import org.lwjgl.glfw.GLFWvidmode;
+import org.lwjgl.opengl.GL11;
+import org.lwjgl.opengl.GLContext;
+import org.lwjgl.system.MemoryUtil;
+
+public class App {
+    private static final Logger log = Logger.getLogger("Main Log");
```
Comment (ynagasaki) [inline src/main/java/com/github/reisnera/gameboylfb/App.java]: all-caps static final vars.

convention for loggers is MyClass.class

File: src/main/java/com/github/reisnera/gameboylfb/App.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    private GLFWErrorCallback errorCallback;
+    private GLFWKeyCallback keyCallback;
+    private long window;
+    private int WIDTH = 160;
+    private int HEIGHT = 144;
```
Comment (ynagasaki) [inline src/main/java/com/github/reisnera/gameboylfb/App.java]: static final

File: src/main/java/com/github/reisnera/gameboylfb/App.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        // Enable v-sync
+        glfwSwapInterval(1);
+
+        // Make the window visible
+        glfwShowWindow(window);
+        //GL.createCapabilities(true); // Load LWJGL's current context without deprecated functions
+        GLContext.createFromCurrent(); // use this line instead with the 3.0.0a build
+
+        // Clear buffer to Game Boy off-state display color
+        GL11.glClearColor(0.605f, 0.734f, 0.059f, 1.0f);
```
Comment (ynagasaki) [inline src/main/java/com/github/reisnera/gameboylfb/App.java]: so many magic numbers...

File: src/main/java/com/github/reisnera/gameboylfb/GameBoyAppLauncher.java [hunk 4]
```diff
@@ -20,6 +20,7 @@
 	static GameBoyRom rom = null;
 	static GameBoyMemory mem = null;
 	static GameBoyCpu cpu = null;
+	static App video = null;
```
Comment (ynagasaki) [inline src/main/java/com/github/reisnera/gameboylfb/GameBoyAppLauncher.java]: Just make all these guys locals to main...

Comment (reisnera) [inline src/main/java/com/github/reisnera/gameboylfb/GameBoyAppLauncher.java]: Well they're gonna be needed in the run() method in that class... The overall structure of the prog isn't final though, and certainly not the GameBoyAppLauncher class specifically.

**Pull request closed without merging**